doc: Updated KRKNWK-16728 known issue content (#11663)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

        return aReadHandler.SetReportingIntervals(exampleMaxInterval);
     }

>   The class implementation could be registered in your application code in a following way:

Reviewer: ```suggestion
  The class implementation can registered in your application code in the following way:
```

  .. code-block::

      #include <app/InteractionModelEngine.h>

      SubscriptionApplicationCallback myClassInstance;

      chip::app::InteractionModelEngine::GetInstance()->RegisterReadHandlerAppCallback(&myClassInstance);

>   You can use an ``ICDUtil`` module implementation that was introduced in the |NCS| v2.4.0 as a reference.

Reviewer: ```suggestion
  You can use the ``ICDUtil`` module implementation that was introduced in the |NCS| v2.4.0 as a reference.
```
